src/blocks/MoonmailConnector.tsx

```diff
@@ -0,0 +1,78 @@
+import useConnections from './hooks/connections'
+import useProps from './hooks/props'
+
+import { ArgTuple, ComposableProps } from './types'
+
+export const options = {
+  actions: {
+    POSTsuccess: {
```

Comment:
I think prefixing `POST` to these actions is not a great format. I can't think of a use case where these actions are only called from specific requests.

Reply:
Fixed, also alphabetized actions to help prevent future merge conflicts